core/templates/pages/learner-dashboard-page/modal-templates/remove-activity-modal.component.ts

```diff
@@ -51,6 +51,11 @@ export class RemoveActivityModalComponent implements OnInit {
         .LEARNER_DASHBOARD_SUBSECTION_I18N_IDS
         .COLLECTIONS) {
       activityType = constants.ACTIVITY_TYPE_COLLECTION;
+    } else if (this.subsectionName ===
+      LearnerDashboardPageConstants
+        .LEARNER_DASHBOARD_SUBSECTION_I18N_IDS
+        .LEARN_TOPIC) {
+      activityType = constants.ACTIVITY_TYPE_LEARN_TOPIC;
```

Comment:
Isn't this the same as 'remove current goal' which is added below?

Reply:
I am not sure I understand this. Here we define the activityType which is learntopic (because in future we will have a mastertopic) and below we define the removeActivityUrlPrefix.

Reply:
Oh ok!